docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,283 @@
+      Policies:
+          Readers:
+              Type: Signature
+              Rule: "OR('Org1MSP.admin', 'Org1MSP.peer', 'Org1MSP.client')"
+          Writers:
+              Type: Signature
+              Rule: "OR('Org1MSP.admin', 'Org1MSP.client')"
+          Admins:
+              Type: Signature
+              Rule: "OR('Org1MSP.admin')"
+          Endorsement:
+              Type: Signature
+              Rule: "OR('Org1MSP.peer')"
+
+      # leave this flag set to true.
+      AnchorPeers:
+          # AnchorPeers defines the location of peers which can be used
+          # for cross org gossip communication.  Note, this value is only
+          # encoded in the genesis block in the Application section context
+          - Host: peer0.org1.example.com
+            Port: 7051
+  ```  
+
+  - The `Name:` field is an informal name used to identify the organization.
+
+  - The `ID:` field is the organization's MSP ID. The MSP ID acts as a unique identifier for your organization, and is referred to by channel policies and is included in the transactions submitted to the channel.
+
+  - The `MSPDir:` is the path to an MSP folder that was created by the organization. The `configtxgen` tool will use this MSP folder to create the channel MSP. This MSP folder needs to contain the following information, which will be transferred to the channel MSP and stored in the channel configuration:
+    - A CA root certificate that establishes the root of trust for the organization. The CA root cert is used to verify if an application, node, or administrator belongs to a channel member.
+    - A root cert from the TLS CA that issued the TLS certificates of the peer or orderer nodes. The TLS root cert is used to identify the organization by the gossip protocol.
+    - If Node OUs are enabled, the MSP folder needs to contain a `config.yaml` file that identifies the administrators, nodes, and clients based on the OUs of their x509 certificates.
+    - If Node OUs are not enabled, the MSP needs to contain an admincerts folder that contains the signing certificates of the organizations administrator identities.
+
+    The MSP folder that is used to create the channel MSP only contains public certificates. As a result, you can build the MSP folder locally, and then pass the MSP to the organization that is creating the channel in an out of band operation.
+
+  - The `Policies:` section is used create a set of signature policies associated with each organization. We will discuss these policies in more detail when we discuss [Channel policies](#channel-policies).
+
+  - The `AnchorPeers:` field is used to identify the lead peers for communication using gossip. While an organization can select their anchor peers on the channel for the first time using the `configtxgen` tool, it is recommended that each organization set anchor peers by using the `configtxlator` tool to [update the channel configuration](create_channel.html#set-anchor-peers).
+
```

Comment:
Do we need to clarify that anchor peers are used to for inter organization communications via the gossip protocol? At least one anchor must be configured per organization.

Reply:
The `AnchorPeers:` field is used to identify the lead peers for inter organization communication using gossip. It is recommended that organizations select at least one anchor peer.